src/test/java/io/personium/jersey/engine/test/ScriptTest.java

```diff
@@ -473,6 +474,32 @@ public final void requireNoFileTest() {
         callService("requireNoFile.js");
     }
 
+    /**
+     * box install test
+     */
+    @Test
+    public final void barInstallTest() {
+        if (!isServiceTest) return;
+
+        try {
+            InputStream is = ClassLoader.getSystemResourceAsStream("testBar.bar");
+
+            putResource("testBar.bar", "application/zip", is);
+            
+            try {
+                callService("boxInstall.js");
```

Comment:
Can you rename the file?

```suggestion
                callService("boxInstallation.js");
```